.fluence/aqua-dependencies/package.json

```diff
@@ -1,9 +1,9 @@
 {
   "dependencies": {
-    "@fluencelabs/aqua-ipfs": "0.5.28",
-    "@fluencelabs/installation-spell": "0.6.4",
-    "@fluencelabs/spell": "0.6.5",
-    "@fluencelabs/aqua-lib": "0.9.0",
+    "@fluencelabs/aqua-ipfs": "0.5.30",
+    "@fluencelabs/installation-spell": "0.6.10",
+    "@fluencelabs/spell": "0.6.10",
+    "@fluencelabs/aqua-lib": "0.9.1",
     "@fluencelabs/registry": "0.9.4"
   }
 }
```

Comment:
```suggestion
}

```

Reply:
this file is generated with fluence cli and this behaviour should be fixed in cli)